frontend/src/components/organisms/ProfileForm.tsx

```diff
@@ -58,16 +53,6 @@ const ProfileForm = ({ userDetails }: ProfileFormProps) => {
         return "User with this email has been disabled.";
       case "user-not-found":
         return "There is no user with this email address.";
```

Comment:
Are any of these errors still needed? I don't think so

Reply:
The argument of `setErrorMessage` after updating the profile should change to the default "something went wrong, try again"

Reply:
Just to confirm, I should remove user-disabled and user-not-found right? And for the default message, do you want us to change from "Something went wrong. Please try again" to "Something went wrong, try again."?

Reply:
The errors listed here are errors gotten from firebase, since you have moved firebase password reset to another component, I don't imagine they will be useful here. However, you  have to be sure removing it altogether does not break anything else. e.g we still need error and success states for changing the user's profile in the db